marklogic-client-api/src/main/java/com/marklogic/client/impl/DatabaseClientPropertySource.java

```diff
@@ -329,61 +271,124 @@ private DatabaseClientFactory.SSLHostnameVerifier determineHostnameVerifier() {
 	 * X509TrustManager.
 	 *
 	 * @param authType used for applying "default" as the SSL protocol for MarkLogic cloud authentication in
-	 *                            case the user does not define their own SSLContext or SSL protocol
+	 *                 case the user does not define their own SSLContext or SSL protocol
 	 * @return
 	 */
 	private SSLInputs buildSSLInputs(String authType) {
-		SSLContext sslContext = null;
+		X509TrustManager userTrustManager = getTrustManager();
```

Comment:
This is where all the changes start. This method should be a lot easier to understand now.